api/v1alpha1/flowcollector_types.go

```diff
@@ -93,6 +96,11 @@ type FlowCollectorGoflowKube struct {
 	// ImagePullPolicy is the Kubernetes pull policy for the image defined above
 	ImagePullPolicy string `json:"imagePullPolicy,omitempty"`
 
+	//+kubebuilder:default:=""
+	// Namespace  where goflowkube pod is going to be deployed.
+	// If empty, the namespace of the operator is going to be used
+	Namespace string `json:"namespace,omitempty"`
```

Comment:
I wasn't thinking there would be two separate fields for goflow-kube and plugin's namespaces, do you think there can be a real need? I'm asking, because it's one more field to configure for the user, and the easier it is to configure, the better is the user experience...

Reply:
I thought the CRD would be more readable with namespace under each section than at the top level.

About the needs, what exactly is the needs of this task ? I mean, if we have need to run kubeflow and the plugin in a different namespace than the operator (while there can only be one kubeflow per operator), we may have needs to run both in differents namespaces.

Reply:
My bad, the story isn't clear about that. The initial need came after discussing with the operator framework team and I was told it could be seen as a bad practice to deploy stuff in the operator's namespace. Think about openshift for instance, where (I think) there a unique namespace where many operators are deployed, but we don't want this namespace to be polluted by workloads that are not operators themselves.
I should have made it clear in the task description, sorry about that.

But then, I don't think we need this level of detail of deploying each component in a separate namespace.

Reply:
Ok, thanks for the clarifications.